benchmark/huffpost_kaggle/data_generation.py

```diff
@@ -0,0 +1,129 @@
+import json
+import shutil
+from pathlib import Path
+from typing import Annotated
+
+import pandas as pd
+from sklearn.model_selection import train_test_split
+import typer
+from torch.utils.data import Dataset
+from tqdm import tqdm
+from modyn.const.types import TimeResolution
+from benchmark.utils.time_resolution_binning import (
+    bin_dataframe_wrt_time_resolution,
+    df_to_csv_with_timestamp,
+    HELP_RESOLUTION,
+)
+
+
+class HuffpostKaggleDownloader(Dataset):
```

Comment:
I acknowledge that there is some degree of code duplication originating from the fact that `huffpost` and `arxiv` have similar data formats and are sourced from kaggle. I think that's justified though as the scripts should be independent. (Changing one shouldn't break anything in the other).

Reply:
yeah that's totally fine here